src/custom-element.js

```diff
@@ -540,6 +540,16 @@ function createBaseCustomElementClass(win) {
         // an unfortunate trade off, but it seems faster, because the DOM
         // operations themselves are not free and might delay
         Services.timerFor(toWin(this.ownerDocument.defaultView)).delay(() => {
+          const TAG = this.tagName;
+          if (!this.ownerDocument) {
+            dev().error(TAG, 'preconnectCallback called on element without ' +
+                'owner docuent');
```

Comment:
Nit: "document". Also we can save a few bytes, e.g. `'preconnectCallback without ownerDocument.'`

Reply:
Done.